### ISSUE TITLE
HeadLess Client  and Task Force Server conflict ("Task force is not running on the server").

### ISSUE THREAD

**Issue by ChristopheTdn** [michail-nikolaev/task-force-arma-3-radio#906]
My config is a dedicated Server and a headless client configured on the same machine (host 
dedicated).

I connect to the server with my own PC

When i start a mission with Headless client and i try to assign some Unit to HC, i got a message that Task force is not running on the server.

After 10 secondes. The message dissapear and i can take Radio and use TFAR. But, my presset frequency  are not valid.

The autoassign Radio on start don t work. In fact, all setting are resetted to default.

If i  don't use HC on the mission, TFAR works like a charm.

**Comment by jonpas** [comment]
We are running 3 Headless Clients on the same (very powerful) machine as the server and never had any issues with TFAR.

**Comment by ChristopheTdn** [comment]
Thx to jour report.

What script do you use to assign IA to the headless client ?

Do you determine radio frequency fort side with script ?

Do you impose same frequency for a side ?

**Comment by jonpas** [comment]
> What script do you use to assign IA to the headless client?
We use http://forums.bistudio.com/showthread.php?170539-AI-Caching-and-Distribution-System for AI distribution (with disabled caching).

> Do you determine radio frequency fort side with script?
No.

> Do you impose same frequency for a side?
Yes, in the userconfig which is synchronized with our modpack over ArmA3Sync.

**Comment by ChristopheTdn** [comment]
thx for your feedback. I give it a try and come back to tell my progress ;-)

**Comment by ChristopheTdn** [comment]
Well, after many test, i solved my problem.

I had to see my method to manage the allocation of the IA
  headlessclient.

In my prior test, the script was running at the start (using a modded script with post init = 1)

I disabled Postinit and just call the script a the end of my Init.sqf only for Server. 

No more server error.

All my source here : http://server.clan-fsf.fr/redmine/projects/fsf_headlessclient/repository
